include/saber/geometry/point.hpp

```diff
@@ -38,6 +39,11 @@ class Point
     constexpr Point& operator*=(const Point& inPoint);
     constexpr Point& operator/=(const Point& inPoint);
 
+private:
+    constexpr bool IsEqual(const Point& inPoint) const;
+    friend constexpr bool operator==<Point>(const Point& inLHS, const Point& inRHS);
+    friend constexpr bool operator!=<Point>(const Point& inLHS, const Point& inRHS);
```

Comment:
Ok, right. I remember...
`operator==`, `operator!=` are `friend`, because they want to call "private" `IsEqual()`.
👍